src/pages/resultsView/pathwayMapper/ResultsViewPathwayMapper.tsx

```diff
@@ -0,0 +1,190 @@
+                                  this.storeForAllData!.patients.result,
+                                  this.storeForAllData!.coverageInformation.result,
+                                  this.storeForAllData!.sequencedSampleKeysByGene.result!,
+                                  this.storeForAllData!.sequencedPatientKeysByGene.result!,
+                                  this.storeForAllData!.selectedMolecularProfiles.result!,
+                                  alterationData,
+                                  false,
+                                  this.alterationFrequencyData,
+                                  this.props.store.genes.result!);
+                
+            });
+
+            this.addGenomicData(this.alterationFrequencyData);
+            // Toasts are removed with delay
+            setTimeout(() => {this.activeToasts.forEach(tId => {toast.dismiss(tId);});}, 1000);
+        }
+
+        // Call to this.remoteGenes.result helps remoteData to work.
+        //console.log("this.remoteGenes");
+        //console.log(this.validNonQueryGenes.result);
+        console.log("this.validGenes");
+        console.log(this.validGenes);
+        return(
+
+            <div className="pathwayMapper">
+            <div data-test="pathwayMapperTabDiv" className="cBioMode" style={{width: "99%"}}>
+                <Row>
+
+                    { !this.isLoading ? 
+                    
+                        <React.Fragment>
+                            <OqlStatusBanner style={{marginLeft: "15px"}} store={this.props.store} tabReflectsOql={true}/>
+                                
+                            <PathwayMapper isCBioPortal={true} isCollaborative={false} 
+                                    genes={this.props.store.genes.result as any}
+                                    cBioAlterationData={this.alterationFrequencyData}
+                                    queryParameter={QueryParameter.GENE_LIST}
+                                    oncoPrintTab={ResultsViewTab.ONCOPRINT}
+                                    changePathwayHandler={this.changePathwayHandler}
+                                    addGenomicDataHandler={this.addGenomicDataHandler}
+                                    tableComponent={PathwayMapperTable}
+                                    validGenes={this.validGenes}
+                                    toast={toast}/>
+                            <ToastContainer/>
+                        </React.Fragment>
+                    :   <LoadingIndicator isLoading={true} size={"big"} center={true}/>
+                    }
+                </Row>
+            </div>
+            </div>);
+    }
+
+    @computed get storeForAllData(){
+        if(this.validNonQueryGenes.isComplete && this.validNonQueryGenes.result.length > 0){
+            const tId = toast("Alteration data of genes not listed in gene list might take a while to load!", {autoClose: false, position: "bottom-left"});
```

Comment:
Generally, if the line doesn't fit the review text area, then you should probably break it:

```suggestion
            const tId = toast("Alteration data of genes not listed in gene list might take a while to load!", 
                {autoClose: false, position: "bottom-left"});
```